packages/components/organisms/f-registration/src/tenants/en-NZ.js

```diff
@@ -1,3 +1,61 @@
 export default {
-    locale: 'en-NZ'
+    locale: 'en-NZ',
+
+    navLinks: {
+        termsAndConditions: {
+            prefix: 'By creating an account you agree to our ',
+            suffix: '.',
+            text: 'Terms and Conditions',
+            url: '/info/terms-and-conditions'
+        },
+        privacyPolicy: {
+            prefix: 'Please read our ',
+            text: 'Privacy Policy',
+            url: '/info/privacy-policy'
```

Comment:
Looks like this should be `/privacy-policy`

Reply:
`/privacy-policy` redirects to `/info/privacy-policy`